compiler.l

```diff
@@ -548,7 +548,7 @@
 (target js: (set (get global 'require) require))
 (target js: (define run eval))
 
-(target lua: (define load1 load))
+(target lua: (define load1 (or loadstring load)))
```

Comment:
Yes, if `loadstring` exists, then `load1` should use that. Otherwise it should use `load`.